src/javascripts/views/transactionView.js

```diff
@@ -0,0 +1,66 @@
+import { validateTransaction } from "../helpers/validationForm";
+
+class TransactionView{
+    constructor(){
+        this.transactionElement = document.querySelector(".transaction");
+        this.transactionBodyElement = document.querySelector(".transaction-body");
+        this.buttonSaveElement = document.querySelector(".btn-save");
+    }
+
+    //----- RENDERING -----//
+    renderPopup(){
+        this.transactionElement.classList.add("transaction-active");
+    }
+
+    closePopup(){
+        this.transactionElement.classList.remove("transaction-active");
+    }
+
+    //----- EVENT LISTENER -----//
+    addEventAddTransaction = (addTransaction) => {
+        this.buttonSaveElement?.addEventListener("click", async(e) => {
+            e.preventDefault();
+
+            const transaction = {
+                category: this.transactionBodyElement.category.value,
+                date: this.transactionBodyElement.date.value,
+                note: this.transactionBodyElement.note.value,
+                outflow: this.transactionBodyElement.outflow.value
+            }
+
+            const errors = validateTransaction(transaction);
+            const errorElements = document.querySelectorAll(".error-date, .error-category, .error-amount, .error-note");
```

Comment:
Instead of using specific class for each element, you can use common class. Then you can find all, and clean message

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!